content-services/content-sources-backend
# Fixes 2270: prevent out of order migrations

## Summary

1.  Creates a new file ./db/migrations.latest which holds the timestamp of the latest migration
2. on db-migrate-up, the command errors out if the value in   ./db/migrations.latest doesn't match the latest in migration in ./db/migrations
3.  `go run ./cmd/dbmigrate/go.mod  new my_migration'  will auto-populate migrations.latest with the newest timestamp
4. The result is that prs should create conflicts on merge if they both contain migrations, ensuring that we never commit any that are out of order.

## Testing steps

Try generating a new migration without updating migrations.latest.  Try to migrate up (you should get an error).
